config/config_test.go

```diff
@@ -261,3 +265,42 @@ streams:
 		s.Equal(test.expectedEksActivatedRules, c.Streams[0].RuntimeCfg.ActivatedRules.CisEks)
 	}
 }
+
+func (s *ConfigTestSuite) TestAWS() {
```

Comment:
Why not edit `TestNew`?

The convention that we usually use is the `TestFunctionName` - like the following suggests https://ieftimov.com/posts/testing-in-go-naming-conventions/#function-naming-conventions

If you found it better to have its own function I don't mind, but maybe edit the test name to

`TestNewConfigFrom` `TestNewConfigFromAwsConfig` or something similar

Reply:
Sounds right